### ISSUE TITLE
getResourceLinks function

### ISSUE THREAD

**Issue by KierynAnnette** [cloudcreativity/laravel-json-api#643]
Looks like this function can no longer be used to override the high level links in the schema. Has this been changed to something else?

Adding:

`public function getResourceLinks(mixed $resource): iterable
    {
        return [];
    }`

no longer removes the high level links in the response.

**Comment by lindyhopchris** [comment]
@KierynAnnette thanks for raising this.

I've attempted to fix on #644 

Can you give it a go and let me know if it works?

```bash
composer require "cloudcreativity/laravel-json-api:dev-issue643"
```

**Comment by KierynAnnette** [comment]
@lindyhopchris  I am only using v5 at the moment. But I applied your changes to that version and I can remove links and add my own. Looks good to me. Thanks

**Comment by lindyhopchris** [comment]
Thanks! I've merged this into v6 and tagged `v6.1.0` so whenever you upgrade your Laravel version you'll be able to get these changes.
